benchmarks/plots-generators/generate-flatten-merge-flow-plots.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib.ticker import FormatStrFormatter
+
+inputFile = "jmh-result.csv"
+outputFile = "flatten-merge-plots.svg"
```

Comment:
I would suggest the following convention:
benchmark = XxxYyyZzzBenchmark
script = generate_plots_xxx_yyy_zzz.py
plot = xxx_yyy_zzz.png